tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -6076,7 +6077,7 @@ function bar(/* Comment */ &$a /* Comment */) {}
 
 function foo(
     /* Comment */ $a,
-    /* Comment */ array /* Comment */ $b /* Comment */ = /* Comment */ [] /* Comment */
+    /* Comment */ array $/* Comment */ b /* Comment */ = /* Comment */ [] /* Comment */
```

Comment:
This is also a bug, that's why tests with AST_COMPARE are failing.

Reply:
@czosel hm, you are right